modules/cluster-logging-curator-tolerations.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-curator.adoc
+
+[id="cluster-logging-curator-tolerations_{context}"]
+= Using tolerations to control the Curator pod placement
+
+You can control which node the Curator pod runs on
+using tolerations on the pod.
+
+You apply tolerations to the Curator pod through the Cluster Logging Custom Resource (CR)
+and apply taints to a node through the node specification. A taint on a node 
+instructs the node to repel all pods that do not tolerate the taint.
+
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+.Procedure
+
+. Use the following command to add a taint to a node where you want the Curator pod scheduled:
```

Comment:
Should we add a step to view the available nodes and identify one to taint?

 s/the Curator pod scheduled/to schedule the Curator Pod